src/rtsp_player_controller.cc

```diff
@@ -489,8 +489,10 @@ void RTSPPlayerController::calculateAudioLevel(AVFrame* input_frame, AVSampleFor
 
 		for (int i = 0; i < nb_samples; i++) {
 			sample = (char)buff16[i];
-			sample = sample / 128.0; //Normalized to (+/- 1.0)
-			sum += (sample * sample);
+			//sample = sample / 128.0; //Normalized to (+/- 1.0)
+			//sum += (sample * sample);			
```

Comment:
any reason to keep these lines in? I'd suggest removing them or adding comments as to why they are important to leave in.